test/SlowTests/Server/Documents/ETL/EtlTimeSeriesTests.cs

```diff
@@ -0,0 +1,1498 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Google.Protobuf.WellKnownTypes;
+using Raven.Client;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Commands.Batches;
+using Raven.Client.Documents.Operations.Attachments;
+using Raven.Client.Documents.Operations.Counters;
+using Raven.Client.Documents.Operations.ETL;
+using Raven.Client.Documents.Session;
+using Raven.Client.Documents.Session.TimeSeries;
+using Raven.Server.Config;
+using Raven.Server.Documents.ETL;
+using Raven.Tests.Core.Utils.Entities;
+using Sparrow.Json.Parsing;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Server.Documents.ETL
+{
+    public class EtlTimeSeriesTests : EtlTestBase
+    {
+        private const int _waitInterval = 1000;
+
+        private abstract class CalculatorTestDataBase : IEnumerable<object[]>
+        {
+            protected const string _script = @"
+loadToUsers(this);
+
+function loadTimeSeriesOfUsersBehavior(doc, counter)
+{
+    return <return>;
```

Comment:
??

Reply:
Just a mark for later to replace by a specific test scenario